### ISSUE TITLE
Regression: Localized name/descriptions don't work properly

### ISSUE THREAD

**Issue by ngyikp** [greasemonkey/greasemonkey#2102]
Test user script:

```javascript
// ==UserScript==
// @name        Name (English)
// @name:fr     Name (French)
// @namespace   ngyikp
// @version     1.0
// @description Description (English)
// @description:fr Description (French)
// @include http://example.com
// ==/UserScript==
```

On Greasemonkey 2.2, the name and description shows properly on install:
![english](https://cloud.githubusercontent.com/assets/6135313/6601201/42709c8a-c84f-11e4-8a0e-41001ef122d2.PNG)

But on Greasemonkey 2.3, the name and description is wrong and uses the last translated value instead of falling back to the unlocalized value:
![french](https://cloud.githubusercontent.com/assets/6135313/6601195/37d0ad2e-c84f-11e4-8b8b-8f24e8c440c5.PNG)

Changing the unlocalized values to the bottom seems to be a workaround for this problem, but I'm not sure how other userscript engines are going to behave:
```javascript
// ==UserScript==
// @name:fr     Name (French)
// @name        Name (English)
// @namespace   ngyikp
// @version     1.0
// @description:fr Description (French)
// @description Description (English)
// @include http://example.com
// ==/UserScript==
```

**Comment by janekptacijarabaci** [comment]
Confirmed.

May I suggest this fix:
https://github.com/greasemonkey/greasemonkey/blob/master/modules/parseScript.js#L64

From:
```javascript
      script['_' + data.keyword] = data.value;
```
To:
```javascript
      else {
        if ((data.keyword == 'description')
            && (script['_' + data.keyword] == ''))
            script['_' + data.keyword] = data.value;
        if ((data.keyword == 'name')
            && ((script['_' + data.keyword] == 'user-script')
            || (script['_' + data.keyword] == name)))
            script['_' + data.keyword] = data.value;
      }
```

**Comment by ngyikp** [comment]
About my workaround, I just thought of duplicating the default strings as the "en" locale at the bottom, it looks a bit odd, but it shouldn't cause problems with other userscript engines.
```javascript
// ==UserScript==
// @name        Name (English)
// @name:fr     Name (French)
// @name:en     Name (English)
// @namespace   ngyikp
// @version     1.0
// @description    Description (English)
// @description:fr Description (French)
// @description:en Description (English)
// @include http://example.com
// ==/UserScript==
```

**Comment by arantius** [comment]
@janekptacijarabaci  Thanks a ton.  Pull requests for your changes would be even more appreciated!

**Comment by arantius** [comment]
This fix is now included in version 3.1beta1.  Please browse to [Greasemonkey's AMO page](https://github.com/Ventero/greasemonkey/commit/b8a09369482f87d0d9fa8afb39b5f46439eab0c2) and open "Development Channel" near the bottom to install the beta version.  Can you confirm the fix?  Thank you.

**Comment by ngyikp** [comment]
Seems to be fixed, thanks!
